main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func main() {
+	content, readErr := ioutil.ReadAll(os.Stdin)
```

Comment:
You're reading all of the data into memory. What happens if you're running this on a VM w/ 1G RAM, but your in.txt is 2G large?

Reply:
which text file is 2G large?

Reply:
`for i in {1..1000}; do for j in {1..1000}; do for k in {1..1000}; do echo 2 done done done > 2G.txt`

That one.

Reply:
do I have to use read.Bytes again?

Reply:
Exactly.